messages/gettest.md

```diff
@@ -32,6 +32,10 @@ ID of the test run.
 
 Directory in which to store test result files.
 
+# flags.concise.summary
+
+Only display failed test results for human-readable output.
```

Comment:
```suggestion
Display only failed test results in the human-readable output.
```

Reply:
Question: I'm a bit confused by the "for human-readable output" part.  Do you mean that this flag works _only_ if "--result-format human"?  If so, something like this might be better:

Display only failed test results; works with human-readable output only.

Reply:
@jshackell-sfdc That's correct. I can update it to `Display only failed test results; works with human-readable output only.`